udf-compiler/src/test/scala/com/nvidia/spark/OpcodeSuite.scala

```diff
@@ -45,7 +41,7 @@ class OpcodeSuite extends FunSuite {
 
   import spark.implicits._
 
-// Utility Function for checking equivalency of Dataset type
+  // Utility Function for checking equivalency of Dataset type
   def checkEquiv[T](ds1: Dataset[T], ds2: Dataset[T]) : Unit = {
     ds1.explain(true)
```

Comment:
Lets remove the `.explain` and the `.show` in `checkEquiv` to cut down on the output.

Reply:
Removed.